### PR TITLE
*. : Modify README.md and README_zh.md #75

### PR DESCRIPTION
**Pull request by molliezhang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ At present, RadonDB MySQL has supported the deployment of MySQL high availabilit
 
 ## Architecture
 
-- Achieving decentralized selection through the Raft protocol.
-- Synchronize data based on GTID mode through Semi-Sync.
+- Achieving decentralized selection be through the Raft protocol.
```
Comment (andyli029) [inline README.md:15]: selection
->
election

File: README.md [hunk 2]
```diff
@@ -12,8 +12,9 @@ At present, RadonDB MySQL has supported the deployment of MySQL high availabilit
 
 ## Architecture
 
-- Achieving decentralized selection through the Raft protocol.
-- Synchronize data based on GTID mode through Semi-Sync.
+- Achieving decentralized election be through the Raft protocol.
+- Synchronizing data based on GTID mode through Semi-Sync.
+- Supporting high-availability and replication capability through Xenon.
```
Comment (andyli029) [inline README.md:17]: Delete replication capability

File: README.md [hunk 3]
```diff
@@ -12,8 +12,9 @@ At present, RadonDB MySQL has supported the deployment of MySQL high availabilit
 
 ## Architecture
 
-- Achieving decentralized selection through the Raft protocol.
-- Synchronize data based on GTID mode through Semi-Sync.
+- Achieving decentralized election through the Raft protocol.
+- Synchronizing data based on GTID mode through Semi-Sync.
+- Supporting high-availability through Xenon.
```
Comment (hustjieke) [inline README.md:17]: add link `https://github.com/radondb/xenon.git` after `Xenon`

File: README_zh.md [hunk 4]
```diff
@@ -14,6 +14,7 @@
 
 - 通过 Raft 协议实现无中心化选主
 - 通过 Semi-Sync 基于 GTID 模式同步数据
+- 通过 Xenon 提供高可用能力
```
Comment (hustjieke) [inline README_zh.md:17]: add link `https://github.com/radondb/xenon.git` after `Xenon`

**Comment by hustjieke** [comment]
change the the mod of  `README.md` to `-rw-r--r--` like ` README_zh.md`,  currently it can be execute.
```
$ ls -l  README.md README_zh.md 
-rwxr-xr-x  1 gaoriyao  staff  2461  4 28 15:18 README.md
-rw-r--r--  1 gaoriyao  staff  2178  4 28 15:18 README_zh.md
hustjason:radondb-mysql-kubernetes gaoriyao$ 

```

**Comment by molliezhang** [comment]
> change the the mod of `README.md` to `-rw-r--r--` like ` README_zh.md`, currently it can be execute.
> 
> ```
> $ ls -l  README.md README_zh.md 
> -rwxr-xr-x  1 gaoriyao  staff  2461  4 28 15:18 README.md
> -rw-r--r--  1 gaoriyao  staff  2178  4 28 15:18 README_zh.md
> hustjason:radondb-mysql-kubernetes gaoriyao$ 
> ```

Done

**Comment by hustjieke** [comment]
LGTM

**Comment by andyli029** [comment]
LGTM

**Pull request merged**